### DISCUSSION TITLE
トカルトの完成度

### DISCUSSION THREAD

**Discussion by anminmakura** [TUSB/TheUnusualSkyBlock#70] [エリア・建築の完成度・開発状況]
以前はみかん氏が製作していたが、
現状開発順位が低いので、無理に開発する必要がないと思い放置している。
鯖が1.16環境になり次第1.16のブロック等を散りばめたいと思っている。

**Comment by huyuyori** [comment]
![2020-11-17_11 24 50](https://user-images.githubusercontent.com/73624149/99513177-af3d5980-29cd-11eb-9847-3dff42f0dfb3.png)
![2020-11-18_18 33 16](https://user-images.githubusercontent.com/73624149/99513192-b19fb380-29cd-11eb-808e-5be9fc31eb09.png)
1.16化進行中

**Comment by huyuyori** [comment]
![2020-11-19_14 13 45](https://user-images.githubusercontent.com/73624149/99630368-18c87100-2a7d-11eb-9d86-a20e4caf7499.png)
トカルト全域をざっと1.16化 スクショの場所以外は大きく変わらず トカルト城、死霊軍団の建築には一切触れてない

**Comment by anminmakura** [comment]
### 要望
- [x] トカルトの辺境の集落に倉庫がほしい

**Comment by anminmakura** [comment]
今後のトカルトでやることリスト

- [x] ツルトベルデ(トカルト地下)内の複数ある歯車及び建造物をこのように青い氷を配置する
 具体的に言えば air、光源系、水、氷塊と氷ブロック以外のブロックを「青い氷ブロック」に変換する
![青い氷の配置の仕方の構想](https://user-images.githubusercontent.com/39670701/110881288-741bb900-8323-11eb-81d9-23360befd7b5.png)



- [x] ツルトベルデ内のバイオームを「ice_spikes」にする。

![2021-03-11_22 08 55](https://user-images.githubusercontent.com/39670701/110881562-e2607b80-8323-11eb-9313-e4a58216ab7a.png)


- [x] 大小異なる水入りのクレバスをトカルトエリアに32個追加
